jwst/lib/set_telescope_pointing.py

```diff
@@ -237,13 +248,20 @@ def update_wcs(model, default_pa_v3=0., siaf_path=None, engdb_url=None,
         exp_type = model.meta.exposure.type.lower()
     except AttributeError:
         exp_type = None
+    aperture_name = model.meta.aperture.name.upper()
+    if aperture_name is not "UNKNOWN":
+        logger.info("Updating WCS for aperture {}".format(aperture_name))
+        useafter = model.meta.observation.date
+        siaf = _read_wcs_from_siaf(aperture_name, useafter, siaf_path)
+        populate_model_from_siaf(model, siaf)
+
```

Comment:
Should we have an "else" clause here that issues a warning (or at least info) when aperture_name IS unknown?

Reply:
Done